vars/stageStartOpenshiftBuild.groovy

```diff
@@ -27,15 +27,17 @@ def call(def context, def buildArgs = [:], def imageLabels = [:]) {
 }
 
 private void patchBuildConfig(def context, def buildArgs, def imageLabels) {
-  // sanitize commit message
+ // sanitize commit message
   def sanitizedGitCommitMessage = context.gitCommitMessage.replaceAll("[\r\n]+", " ").trim().replaceAll("[\"']+", "")
+  //remove apostrophe in committer name-fix for issue-https://github.com/opendevstack/ods-project-quickstarters/issues/358
```

Comment:
I transferred the issue to the shared library. Could you remove the now no longer applicable reference to the issue in the comment?

Reply:
I have removed the link in the comment